crates/symbolicli/README.md

```diff
@@ -0,0 +1,29 @@
+`symbolicli` is a cli utility that lets you symbolicate native events and minidumps locally.
+
+By default, `symbolicli` will target https://sentry.io/. If you are hosting your own
+Sentry instance, you can override this with the `--url` option or the `url` config setting.
+
+# Basic usage
+```
+symbolicli -o <ORG> -p <PROJECT> --auth-token <TOKEN> <EVENT>
+```
+where
+
+* `<ORG>` and `<PROJECT>` are the organization and project to which the event belongs;
+* `<TOKEN>` is a Sentry authentication token that has access to the project;
+* `<EVENT>` is either a local file (minidump or event JSON) or the ID of an event on the Sentry instance.
+
+# Configuration
+
+`symbolicli` can be configured via the `~/.symboliclirc` config file, written in the TOML format.
```

Comment:
I'd use https://github.com/getsentry/sentry-cli/blob/8267a50608dd5d74b32fbeafaddd53f9257624d0/src/config.rs#L464-L479 from `sentry-cli` and go up the path stack to find the nearest `.symboliclirc` instead of allowing home dir only.